### PR TITLE
merge web help menu with Help menu 

### PR DESCRIPTION
**Pull request by butterw**
As discussed in this [thread](https://forums.triplea-game.org/post/50332) merges the web help menu with Help menu.

### CODE REVIEW

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/TripleAMenuBar.java [hunk 1]
```diff
@@ -30,7 +30,6 @@ public TripleAMenuBar(final TripleAFrame frame) {
       add(new NetworkMenu(watcher, frame));
     }
 
-    add(new WebHelpMenu());
```
Comment (DanVanAtta) [inline game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/TripleAMenuBar.java:33]: Looks like WebHelpMenu.java can be deleted.

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java [hunk 2]
```diff
@@ -43,6 +44,26 @@ public static JMenu buildMenu(final UiContext uiContext, final GameData gameData
...
+        e -> SwingComponents.newOpenUrlConfirmationDialog(UrlConstants.USER_GUIDE));
+    menu.add(hostingLink);
+
+    final JMenuItem warClub = new JMenuItem("TripleA Forum");
+    warClub.setMnemonic(KeyEvent.VK_W);
+    warClub.addActionListener(
+        e -> SwingComponents.newOpenUrlConfirmationDialog(UrlConstants.TRIPLEA_FORUM));
+    menu.add(warClub);
+
+    final JMenuItem donateLink = new JMenuItem("Donate");
```
Comment (DanVanAtta) [inline game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java:61]: A donate menu item in a 'help' menu seems a bit odd. I would expect this is an about menu or something. Let's discuss where it should belong, perhaps 'help' is the least worst place for it, IDK.

**Review by DanVanAtta — CHANGES_REQUESTED** [review]

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java (hunk 2 above)
Comment (butterw) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java:61]: It's the best place for the donate link for now. 

Creating a ```Help > About Triplea``` entry which opens a dialog makes a lot of sense for v3.0. 
The donate and maybe some other links could be moved there, the Triplea version and some artwork could be displayed, etc.

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/TripleAMenuBar.java (hunk 1 above)
Comment (butterw) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/TripleAMenuBar.java:33]: Done in latest commit.

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java (hunk 2 above)
Comment (DanVanAtta) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java:61]: After thinking about it, I'd agree it's a decent enough place. "Donate" is related to "help" (just it's the player helping TripleA vs the other way round).

**Review by DanVanAtta — APPROVED** [review]

**Comment by butterw** [comment]
Code for my AboutTriplea UI, add it to v2.6 if you want:
https://github.com/butterw/ui/blob/main/code/AboutTriplea.java

About...

**Pull request merged**